### ISSUE TITLE
Stoke-like lesions on MRI

### ISSUE THREAD

**Issue by MLK6014** [obophenotype/human-phenotype-ontology#6368]
**Preferred term label:**
Stoke-like lesions on MRI

**Synonyms**

**Definition (free text, please give PubMed ID)**
Stroke-like lesions observed in magnetic resonance imaging (MRI) of the brain

**Parent term (use hpo.jax.org/app)**
Abnormality of the cerebral vasculature HP:0100659

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
This is not a good term because stroke can be associated with a variety of lesions (e.g., ischemic vs hemorrhagic). The label is also an inference ("these lesions look like a stroke but are actually something else") and not an actual phenotypic observation (e.g.,  high T2 signal). If this is coming from the literature or a report and you have no other information, then I think we should close the issue. If you would like to explore better ways of describing these lesions with HPO we can try to work on this section.

**Comment by MLK6014** [comment]
I have forwarded your above comments to Professor John Christodoulou Director of Genetics Brain and Mitochondrial Research group (who are the requesters of this new term) to enable us to gain a further understanding and hopefully move forward with this request.  Below is his response

"_The neurological and imaging features are pretty distinctive, the latter evidenced by the image below.  The MRI findings are neither ischaemic nor haemorrhagic, but are nonetheless real (some refer to this as “metabolic stroke”, as they do not follow a vascular distribution), and evolve over time, as can be seen by the serial images.  This group also reported “A T1-weighted hyperintense cortical signal on MRI compatible with cortical laminar necrosis was seen during the subacute stage of all stroke-like lesions”. 
So, I think a case can be made for this phenotype to be recognised as quite distinct, and when seen it points very much to particular mitochondrial disorders, most notably the common MELAS mutation."_
I have attached the image he references above for your information as well.  Hopefully we can progress from here.

![stroke like lesions](https://user-images.githubusercontent.com/73153740/96959835-7d9eb300-1533-11eb-851b-58e37dc4e880.png)

**Comment by pnrobinson** [comment]
The main issue is that the HPO is used for thousands of different diseases. "Stroke-like lesion" is unambiguous is we know that we are talking about MELAS, but it is not a description of the actual phenotypic feature. Phenotypically, MELAS patients can present with manifestations that are indistinguishable from other kinds of stroke, e.g., aphasia. I am not entirely aware of the phenotypic spectrum of imaging findings in MELAS and other mitochondrial diseases, but to my knowledge if you just look at the MRI image shown above, no neuroradiologist would be able to make the diagnosis of MELAS if they knew nothing of the clinical history. And even if the images above are suggestive of MELAS, I am guessing they are not pathgnomonic, and so calling them "MELAS-like lesions" or something like that could lead to errors in other cases.
 The HPO takes each feature in isolation -- this makes the ontology approach powerful for differential diagnosis!

I am wondering if it is possible to describe the findings more directly.
The HPO needs more detail in this area. Here is the current status
https://hpo.jax.org/app/browse/term/HP:0030890
Looking at the above images, it seems we see
Parietal cortical (?) T1 hyperintensity
Occipital cortical (?) T1 hyperintensity

I am also seeing that there is an increased diffusion-weighted imaging signal (cortical) with increased apparent diffusion coefficient

Could we try to develop some terms in this area to capture these phenotypes? I would love to have your input/collaboration!

**Comment by MLK6014** [comment]
Thank you for your further comments we have looked further at the area you suggested and have found a concept HP:0002401 and have decided to utilise that concept instead.  So will for close this issue for now.  Thanks for your advice
